composer.json

```diff
@@ -29,7 +29,8 @@
     "require-dev": {
         "psr/log": "~1.0",
         "phpcr/phpcr-api-tests": "2.1.4",
-        "symfony/console": "~2.0"
```

Comment:
we provide a command to start the jackrabbit.jar. which is probably not the best idea in the first place. and there is currently no test for it, so we can just as well drop this here.